app/views/modals/_session_timeout_warning.html.slim

```diff
@@ -1,11 +0,0 @@
-#session_timeout_warning.modal.fade tabindex='-1' role='dialog' aria-hidden='true'
```

Comment:
this was legacy html - we don't have or support modals, and the css classes referenced here are no longer relevant. see `app/views/session_timeout/_warning.html.slim` for the reworked html snippet.